From 06d552bc6546d8bacbdf9df512349a6aaeafc5ba Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Thu, 19 Jul 2012 15:06:21 +0300
Subject: [PATCH 256/370] conf/obj.c: add top-level "Threading ..." section

Add conf-lspec-thread section (Threading and Concurrency Model),
which refers to conf{c,d}-lspec-thread.

LogD #490
RB: https://reviewboard.clusterstor.com/r/865/
---
 conf/obj.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/conf/obj.c b/conf/obj.c
index 47a38c8..897afce 100644
--- a/conf/obj.c
+++ b/conf/obj.c
@@ -176,6 +176,7 @@
  * - @ref conf_dlspec_map
  * - @subpage confd-lspec
  * - @ref conf-lspec-state
+ * - @ref conf-lspec-thread
  *
  * <!---------------------------------------------------------------->
  * @subsection conf-lspec-comps Components Overview
@@ -206,6 +207,12 @@
  * - @ref conf-fspec-obj-status
  * - @ref conf-fspec-obj-pinned
  *
+ * <!---------------------------------------------------------------->
+ * @subsection conf-lspec-thread Threading and Concurrency Model
+ *
+ * - @ref confc-lspec-thread "confc"
+ * - @ref confd-lspec-thread "confd"
+ *
  * <hr> <!------------------------------------------------------------>
  * @section conf-conformance Conformance
  *
-- 
1.8.3.2

